scout/parse/case.py

```diff
@@ -41,6 +41,41 @@ def get_correct_date(date_info):
     return datetime.datetime.now()
 
 
+def parse_custom_images(config_data):
+    """Parse information on custom images assigned to the case."""
+    VALID_IMAGE_SUFFIXES = ["svg", "png", "jpeg"]
+    LOG.debug("Parse custom images")
+
+    # sort custom image sections
+    parsed_sections = {}
+    for section_name, images in config_data.get("custom_images", {}).items():
+        parsed_images = []
+        for image in images:
+            # skip entries that are not recognized as image on suffix
+            path = Path(image["path"])
+            if path.suffix in VALID_IMAGE_SUFFIXES:
+                LOG.warning(f"Image: {path.name} is not recognized as an image, skipping")
+                continue
+            # load image file to memory
+            with open(image["path"], "rb") as image_file:
+                parsed_images.append(
+                    {
+                        "title": image["title"],
+                        "description": image["description"],
+                        "data": bytes(image_file.read()),
+                        "width": image.get("width"),
+                        "height": image.get("height"),
+                        "format": "svg+xml" if path.suffix[1:] == "svg" else path.suffix[1:],
+                    }
+                )
+        # store parsed section
+        if len(parsed_images) != 0:
```

Comment:
Maybe replace with `>0` to avoid negative comparisons, a little bit easier to read.

Reply:
sure. Length can never be a negative value though